subt/zmq-subt-x2.json

```diff
@@ -121,6 +124,7 @@
               ["rosmsg.depth", "detector.depth"],
 
               ["localization.pose3d", "app.pose3d"],
```

Comment:
```suggestion
```

Reply:
As mentioned in my other comment: Only if you are willing to break X2-type robots.

Reply:
Uff. So we have two connections to `app.pose3d` at the same time - one comming from `localization.pose3d` (that one is already corrected for origin offset) and another one coming from `rospy.pose3d` (that one is raw and needs correction) and we are trying to rely on the fact that on X2 `rospy.pose3d` does not publish and on Freya localization.pose3d does not publish. 

Is that correct observation?

Reply:
yes, it makes sense - we have to choose only one source of pose3d - there is no "arbitration" on `app` side